### ISSUE TITLE
Schema docs reference outdated methods

### ISSUE THREAD

**Issue by knivets** [encode/django-rest-framework#6887]
https://www.django-rest-framework.org/api-guide/schemas/

Specifically the block that describes how to override `AutoSchema` references an undefined method `get_link()`:

```
    from rest_framework.views import APIView
    from rest_framework.schemas.openapi import AutoSchema

    class CustomSchema(AutoSchema):
        def get_link(...):
            # Implement custom introspection here (or in other sub-methods)

    class CustomView(APIView):
        """APIView subclass with custom schema introspection."""
        schema = CustomSchema()
```

**Comment by ajyong** [comment]
I believe this method does exist as part of the `AutoSchema` class, I can't seem to get it to be considered an undefined method since [DRF 3.7](https://github.com/encode/django-rest-framework/blob/3.7.0/rest_framework/schemas/inspectors.py#L149).  Could it be the version of DRF that you have installed?

**Comment by kevin-brown** [comment]
Closing this off since it appears to have been corrected in #7013 and not automatically closed when it was merged.
